### PR TITLE
Reformat jit source base and turn on formatting jobs for PR

### PR DESCRIPTION
**Pull request by adiaaida**

### CODE REVIEW

**Comment by adiaaida** [comment]
@dotnet-bot test Windows formatting please

**Comment by adiaaida** [comment]
@dotnet-bot test Windows_NT formatting please

**Comment by adiaaida** [comment]
@dotnet-bot test Ubuntu formatting please

**Comment by adiaaida** [comment]
It looks like the Ubuntu formatting job was broken by the CLI upgrade Jan did yesterday. I'm tracking this down so that we can get this checked in.

**Comment by adiaaida** [comment]
@dotnet-bot test Ubuntu formatting please
@dotnet-bot test Windows_NT formatting please

**Comment by adiaaida** [comment]
@dotnet-bot test Ubuntu formatting please
@dotnet-bot test Windows_NT formatting please

**Comment by adiaaida** [comment]
@dotnet-bot test Ubuntu formatting please
@dotnet-bot test Windows_NT formatting please

**Comment by adiaaida** [comment]
@dotnet/jit-contrib PTAL

**Comment by adiaaida** [comment]
NOTE: This change will turn on the formatting checks for all coreclr PRs (but only runs on jit code).
@mmitche FYI

File: src/jit/codegenarm64.cpp [hunk 1]
```diff
@@ -6802,7 +6802,8 @@ void CodeGen::genPutArgStk(GenTreePtr treeNode)
                 var_types type = nextArgNode->TypeGet();
                 emitAttr  attr = emitTypeSize(type);
 
-                // Emit store instructions to store the registers produced by the GT_FIELD_LIST into the outgoing argument
+                // Emit store instructions to store the registers produced by the GT_FIELD_LIST into the outgoing
+                // argument
                 // area
```
Comment (JosephTremoulet) [inline src/jit/codegenarm64.cpp]: Bad line wrap

Comment (adiaaida) [inline src/jit/codegenarm64.cpp]: Good catch.

**Comment by pgavlin** [comment]
LGTM

**Pull request merged**